spec/services/users/delete_spec.rb

```diff
@@ -151,7 +151,7 @@ def create_associations(names)
       described_class.call(user)
       aggregate_failures "associations should exist" do
         kept_associations.each do |kept_association|
-          expect { kept_association.reload }.not_to raise_error, kept_association
+          expect { kept_association.reload }.not_to raise_error
```

Comment:
@rhymes I noticed that you made this change a while ago. Were you hoping to reload those instances? I removed it because RSpec would generate a lot of noisy logs.

```
WARNING: ignoring the provided expectation message argument (#<FeedbackMessage id: 31, affected_id: 10011, category: "rude or vulgar", created_at: "2021-03-30 19:21:47", feedback_type: "abuse-reports", message: "Mixtape distillery normcore chicharrones microdosi...", offender_id: nil, reported_url: "/", reporter_id: nil, status: "Open", updated_at: "2021-03-30 19:21:47">) since it is not a string or a proc.
......
```